tests/test_checkout.py

```diff
@@ -62,6 +62,24 @@ def test_clear_shipping_method(checkout, shipping_method):
     assert not checkout.shipping_method
 
 
+def test_last_change_update(checkout):
+    last_change = checkout.last_change
+
+    checkout.note = "Sample note"
+    checkout.save()
+
+    assert checkout.last_change != last_change
```

Comment:
Mamy we should use `freezgun` to check last_change value instead check not equals?